clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3330,6 +3330,105 @@ static void handleUseStallEnableClustersAttr(Sema &S, Decl *D,
   handleSimpleAttribute<SYCLIntelUseStallEnableClustersAttr>(S, D, Attr);
 }
 
+// Handles disable_loop_pipelining attribute.
+static void handleSYCLIntelFPGADisableLoopPipeliningAttr(Sema &S, Decl *D,
+                                                         const ParsedAttr &A) {
+  S.CheckDeprecatedSYCLAttributeSpelling(A);
+
+  // [[intel::disable_loop_pipelining] and [[intel::initiation_interval()]]
+  // attributes are incompatible.
+  if (checkAttrMutualExclusion<SYCLIntelFPGAInitiationIntervalAttr>(S, D, A))
```

Comment:
The documentation says this attribute is incompatible with `max_interleaving, speculated_iterations,
max_concurrency, initiation_interval, or ivdep`, so I think there are some mutual exclusion checks missing.

Reply:
they are already available as loop attribute, we do not have any max_interleaving, speculated_iterations,or ivdep function attribute.

Reply:
Ah, thank you for pointing that out, that makes sense.

Reply:
I will update documentation.